docs/incident_attributes.rst

```diff
@@ -360,7 +360,7 @@ The :class:`Event <Event>` object in this scenario will have the following attri
 
 Canary Disconnected
 ======================
-Event is generated when a Canary does not contact the console within a defined time period.
+An event is generated when a Canary does not contact the console within a defined time period.
```

Comment:
Either leave it as "Event is .." (it's just a description of the event) or change it to "A Disconnect event is... "